### DISCUSSION TITLE
The link to the training dataset is no longer working

### DISCUSSION THREAD

**Discussion by shiningsword** [Ant-Brain/EfficientWord-Net#35] [General]
This is really great stuff. We were looking at using a tranformer model to further improve the result. Is there any way to get access to your dataset? The link in the readme to the Google drive is no longer working.

**Comment by TheSeriousProgrammer** [comment]
The dataset link is the older model's dataset. The newer one's dataset is a derivative of People Common Speech because of which it is huge, will try to upload it up this weekend

**Comment by TheSeriousProgrammer** [comment]
You can consider using this repo in huggingface datasets https://huggingface.co/datasets/MLCommons/ml_spoken_words/tree/main/data/wav/en
